tools-local/tasks/mobile.tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -219,7 +219,7 @@ public override bool Execute ()
 
         using (var sw = File.CreateText(Path.Join(AppDir, "run-v8.sh")))
         {
-            sw.WriteLine("v8 --expose_wasm runtime.js -- --run " + Path.GetFileName(MainAssembly) + " $*");
+            sw.WriteLine("v8 --expose_wasm runtime.js -- --run " + Path.GetFileName(MainAssembly));
         }
```

Comment:
The purpose of this is to allow running run-v8.sh with arguments.

Reply:
@vargaz ah, I see

Reply:
fixed.